### ISSUE TITLE
[Bug] FPM/Immersive portals interaction

### ISSUE THREAD

**Issue by AlienXtream** [tr7zw/FirstPersonModel#355]
portals with "non conformal" rotation (i.e that "change" the direction you are facing when entered).

in this example i used the immersive protal "rotate y 15" debug stick (found in the "misc" section of the creative menu) to rotate a nether portal 180 degrees. this occurs on both forge and fabric versions of the mod. i shall also be posting this issue to the immersive portals bug tracker and will add a reference to that issue here when thats done
![image](https://user-images.githubusercontent.com/12606809/236706965-26f7ae7d-bdde-47bf-b77d-217c5e444984.png)

**Comment by tr7zw** [comment]
UHHHHHHH. Looks like it's placing a visual second player on the other side/the player being on both sides + with different rotations. I'll be quite honest, that sounds like a bug I don't want to touch.

**Comment by AlienXtream** [comment]
fair enough. its at least documented so if someone else (or even the FPM dev themselves) comes across it and are able to they can fix. its not gamebreaking by any stretch, just a bit weird looking. has no affects on gameplay other than edge cases of it obstructing visibility at times (and you'd really need to be not paying attention for that to be a problem). only reason i even noticed it is because another mod is using IP for a Tardis and in that case the exterior can be any cardinal direction (while the interior obviously stays at a fixed orientation). again, very niche edge cases.
